src/main/kotlin/calculator/main.kt

```diff
@@ -0,0 +1,12 @@
+package calculator
+fun main(args: Array<String>) {
```

Comment:
```suggestion
fun main() {
```
별도의 인자를 받지 않아도 선언 가능합니다!